controller/src/beerocks/master/db/db.cpp

```diff
@@ -2929,6 +2929,11 @@ bool db::set_node_channel_bw(const std::string &mac, int channel, beerocks::eWiF
         }
     }
 
+    LOG(INFO) << "set node " << mac << " to channel=" << channel << ", bw=" << int(bw)
```

Comment:
How often does this happen? We don't want the log to become too clattered.
If you decide to keep this commit, I suggest explaining the scenario in which you felt this print would help so anyone reading this can understand why it is actually needed (me included).

Reply:
It should happen once when receiving a slave join message which initiates the channel selection.
It also happens when sta is connected, and we update it's channel in the node, but IMO we need to remove the channel from sta node since it will always have the same channel as its parent radio.
Please try to understand that I didn't write most of the code here, so I don't always have an answer or explanation of why some change has been done, I can only guess.